Verniana-Jules Verne Studies.js

```diff
@@ -31,51 +30,74 @@
 	<http://www.gnu.org/licenses/>.
 */
 
+// eslint-disable-next-line
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
 
-function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
-function doWeb(doc, url) { return FW.doWeb(doc, url); }
+function detectWeb(doc, _url) {
+	if (doc.querySelector('ul.volume_toc li')) return 'multiple';
+	if (doc.querySelector('.ref_on_top')) return 'journalArticle';
+	return false;
+}
+
+function match(t, r) {
+	let m = t.match(r);
+	return m ? m[1] : '';
+}
+
+function scrape(doc, url) {
+	let item = new Zotero.Item('journalArticle');
+	item.url = url;
+	item.title = ZU.capitalizeTitle(ZU.trimInternal(text(doc, '#content h1') || '')) || 'Verniana — Jules Verne Studies';
+	item.creators = [ZU.cleanAuthor(text(doc, 'div#content p.author a') || text(doc, 'div#content p.author'), 'author')];
```

Comment:
- Is there an example for the first expression, i.e. where the authors are linked? If not, then let us delete this.
- This currently does not handle the case, when there are more than one author, e.g. http://www.verniana.org/volumes/07/Resumes/AlcideMorgaz.html . Can you handle that case as well? And then add the example as another test case.

Reply:
I don't really know -- I just took this logic from the existing translator. I was looking to make the changes as minimal as I could.

I'll look at the multiple-authors case and add it as a test case